### PR TITLE
MDCT-2496: MLR PDF a11y Improvements

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Improving on a couple of items related to PDF accessibility:

- Marking PDF icons as decorative by setting `alt-text` attributes to null
- Matching headings hierarchy from Figma for screen reader traversal
- PDF metadata needs to be in the same hierarchy

### Related ticket(s)

MDCT-2496

---
### How to test

- Create a report
- Export it as PDF

Check for any `a11y` issues related to the description

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx [hunk 1]
```diff
@@ -151,9 +151,6 @@ describe("Test renderModalOverlayTableBody", () => {
       mockReportContext.report.fieldData.program.length + 1
     );
 
-    // Correct status
-    expect(await findByAltText("complete icon")).toBeVisible();
-
```
Comment (ntsummers1) [inline services/ui-src/src/components/export/ExportedModalOverlayReportSection.test.tsx:156]: Should this be removed? Maybe another way to check if this is visible or as a last resort a data-test-id check would work here instead

**Review by ntsummers1 — DISMISSED** [review]
LGTM 👍

File: services/ui-src/src/components/export/ExportedEntityDetailsTableRow.tsx [hunk 2]
```diff
@@ -49,11 +45,18 @@ export const ExportedEntityDetailsTableRow = ({
...
-                  : formField?.props?.label}
+                {!isDynamicField ? (
+                  optional ? (
+                    <Box>
+                      {formFieldInfo.label}
+                      <span className="optional-text"> (optional)</span>
+                    </Box>
+                  ) : (
+                    formFieldInfo.label
+                  )
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/export/ExportedEntityDetailsTableRow.tsx:56]: I appreciate this inlining

**Review by benmartin-coforma — APPROVED** [review]
I have not tested this, but the code looks good

File: services/ui-src/src/components/export/ExportedEntityDetailsOverlaySection.tsx [hunk 3]
```diff
@@ -112,27 +112,27 @@ export function getEntityTableComponents(
...
       <Box key={uuid()}>
         <Box sx={sx.entityInformation}>
           <Heading sx={sx.entityHeading} as="h3" fontSize={"xl"}>
             {entityHeading}
-            <ul>
-              {programInfo.map((field, index) => (
-                <li key={index}>{renderHtml(field)}</li>
-              ))}
-            </ul>
+            {programInfo.map((field, index) => formatProgramInfo(index, field))}
```
Comment (benmartin-coforma) [inline services/ui-src/src/components/export/ExportedEntityDetailsOverlaySection.tsx:135]: Question: Are these fields are coming from user input, from the report template, or from verbiage files?

I think they're user input, but I'm only 80% sure. And if so, then I totally approve this change.

Comment (karla-vm) [reply services/ui-src/src/components/export/ExportedEntityDetailsOverlaySection.tsx:135]: Correct, these are indeed user input!

**Review by benmartin-coforma — DISMISSED** [review]

**Review by ntsummers1 — DISMISSED** [review]

**Review by ntsummers1 — APPROVED** [review]

**Pull request merged**